src/qibolab/serialize.py

```diff
@@ -173,6 +179,13 @@ def dump_runcard(platform: Platform, path: Path):
         path (pathlib.Path): Path that the yaml file will be saved.
     """
 
+    kernels = {}
+    for qubit in platform.qubits.values():
+        if qubit.kernel is not None:
+            kernels[str(qubit.name)] = qubit.kernel
+            qubit.kernel = None
+    Kernels(kernels).dump(Path(__file__).parent / "dummy/kernels.npz")
```

Comment:
```suggestion
    kernels = Kernels()
    for qubit in platform.qubits.values():
        if qubit.kernel is not None:
            kernels[str(qubit.name)] = qubit.kernel
            qubit.kernel = None
    kernels.dump(Path(__file__).parent / "dummy/kernels.npz")
```
Similar to the one above :)

Reply:
Same here probably we can drop `str`.